package.json

```diff
@@ -10,7 +10,8 @@
   "scripts": {
```

Comment:
you can add a 'postinstall' script that will run immediately after npm install finishes. You can add anything that must run to here, for example I added my webpack build to the 'postinstall' script

Reply:
Good to know, thanks!